genai-image-gen-processor/utils/comprehend.py

```diff
@@ -9,7 +11,7 @@ def is_prompt_positive(comprehend_client, prompt, LANGUAGE_CODE):
     print('Sentiment:', sentiment)
     print('Score:', score)
 
-    if sentiment.upper() in ('POSITIVE', 'NEUTRAL'):
+    if sentiment.upper() in (positive_prompt, neutral_prompt):
```

Comment:
As it is a tuple define the tuple in constants and use as it is here, rather than importing individual values

Reply:
Done